api/hooks/sdtdLogs/index.js

```diff
@@ -28,6 +28,9 @@ module.exports = function sdtdLogs(sails) {
         sails.log.info('Initializing custom hook (`sdtdLogs`)');
 
         try {
+          // Make sure there are no lingering jobs
+          // TODO: Once we scale this to multiple processes, this should happen differently
+          await sails.helpers.getQueueObject('logs').empty();
```

Comment:
Ah yes you are correct, my solutions here is short sighted. What was happening was when the app restarts and there is a job in queue from previous run, a new job would be added. Essentially having 2 intervals per server.

Instead of resetting like this, we should check if there is a job in queue already and based on that decide if we add a new one or not